.gitignore

```diff
@@ -277,3 +277,4 @@ lcov.info
 # WindowsInstaller generated files
 /WalletWasabi.WindowsInstaller/ComponentsGenerated.wxs
 /WalletWasabi.WindowsInstaller/*.wixobj
+.ionide/symbolCache.db
```

Comment:
This file is created in my environment by a F# vscode plugin and I need to ignore it because it is annoying.